src/containers/Statistics/Statistics.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { StatsCollateralChart } from './../../components/Stats/StatsCollateralChart/StatsCollateral'
+import { DebtPoolContainer } from './../DebtPoolContainer/DebtPoolContainer'
+import { useSelector } from 'react-redux'
+import { getCollateralStructure, getSyntheticsStructure } from './../../store/selectors/exchange'
+import { Container, Grid } from '@material-ui/core'
+import useStyles from './style'
+
+export const Statistics: React.FC = () => {
+  const colors = [
+    '#FF9494',
+    '#6372BE',
+    '#40BFA0',
+    '#117098',
+    '#BFB665',
+    '#1F70CF',
+    '#936BC7',
+    '#39D3F5',
+    '#DADCF1',
+    '#C76BA2',
+    '#D49347'
+  ]
+  const getCollaterals = useSelector(getCollateralStructure)
+  const getSynthetics = useSelector(getSyntheticsStructure)
+  const classes = useStyles()
+  const CollateralData = Object.values(getCollaterals).map((item, index) => {
```

Comment:
nie wiem, czy w taki sposób będzie się to aktualizować przy każdej zmianie wartości tego, co mapujesz, bo to przypisanie zajdzie chyba tylko raz; przenieś może te mapy tam, gdzie aktualnie używane są te zmienne

Reply:
I noticed on project, $ values of Synthetics changes. Like Redux state